docs/quickstart-docker.md

```diff
@@ -0,0 +1,192 @@
+---
+description: A quick start guide for Featureform with Docker.
+---
+
+# Quickstart (Docker)
+This quickstart uses the standalone containered version of Featureform. It can be used to connect to the same providers 
+as the Kubernetes Hosted version, but lacks the scaling capabilities.
+
+This quickstart will walk through creating a few simple features, labels, and a training from a fraud
+detection dataset using Postgres and Redis.
+
+### Requirements
+
+- Python 3.7+
+- Docker
+
+## Step 1: Install The Featureform CLI
+```shell
+pip install featureform
+```
+
+## Step 2: Pull Containers
+We'll use postgres and redis as providers in this quickstart. We've created a Featureform Postgres container that
```

Comment:
should postgres and redis be capitalized

Reply:
fixed